content/release-10-18-0/edge-10-18-0-bundle/10_18_0_1.md

```diff
@@ -0,0 +1,28 @@
+---
+weight: 59
+title: Release 10.18.0.1
+layout: redirect
+---
+
+The release notes for **Cumulocity IoT Edge release 10.18.0.1** contain important information about all the changes, including new features, enhancements, and fixes, that have been made to both **Cumulocity IoT Edge Appliance VM** and **Cumulocity IoT Edge on Kubernetes**. If you want to know more, you can check out the [What´s New](/release-10-18-0/whatsnew-10-18-0/) section and watch our video [What's new in Cumulocity IoT 10.18](https://www.youtube.com/watch?v=pTouaQbc6SI&list=PLexm-BA6Wrr8AYnu97vpIR7uicD2wsyms).
+
+Additionally, the release notes contain [announcements](/release-10-18-0/edge-10-18-0/#announcements) regarding planned and implemented deprecations that might require some action on your part at some point.
+
+
+You can find the corresponding documentation for **Cumulocity IoT Edge Appliance VM** at https://cumulocity.com/guides/10.18.0/edge/edge-introduction/ and for **Cumulocity IoT Edge on Kubernetes** at https://cumulocity.com/guides/10.18.0/edge-k8s/k8-edge-introduction/.
+
+
+### Cumulocity IoT Edge on Kubernetes
+Cumulocity IoT Edge on Kubernetes has reached General Availability (GA) and is now ready for deployment in production environments.
+
+#### Offline installation
+
+Edge installation on Kubernetes now offers support for installation in an offline or air-gapped environment. This ensures that the required software, Helm Charts and Docker images, are available in the disconnected environment through an [Open Container Initiative](https://opencontainers.org/) (OCI) compliant private registry.
+
+For more information, refer to [Installing the Edge Operator (Offline)](https://cumulocity.com/guides/10.18.0/edge-k8s/installing-edge-on-k8/#air-gapped-installation).
+
+#### Metrics monitoring port
+
+The Edge operator provided Prometheus-compatible metrics endpoint's port is changed from 8443 to 3443 due to port conflict with another core component.
```

Comment:
Above it says "Edge Operator". Please unify the spelling to what is used in the docs

Reply:
Please split this term: "Edge operator provided Prometheus-compatible metrics endpoint's port", it is hard to understand.

Reply:
How does this look?
The Edge operator's Prometheus-compatible metrics endpoint port has been changed from 8443 to 3443 due to a conflict with another core component.